docs/rules/valid-it-test-desc.md

```diff
@@ -0,0 +1,31 @@
+# Check "it()" and "test()" descriptions against a pre-configured regular expression
+
+This rule enforces the "it()" and "test()" descriptions to follow the desired format. 
```

Comment:
`specify` is another alias for `it`.
I’m not saying that we need to implement support for this alias now, but maybe we should rethink the name of the rule to make future additions possible without changing the rule name.
Maybe we can call it just `valid-description` then we could easily add support for e.g. `describe` in the future. What do you think?